themes/digital.gov/layouts/shortcodes/ring.html

```diff
@@ -0,0 +1,9 @@
+{{- $title := (.Get "title") -}}
+
+
+<fieldset class="ring">
+  <legend>
+    {{- $title -}}
+  </legend>
+  {{ .Inner | markdownify }}
+</fieldset>
```

Comment:
Do you expect form inputs here? If it'll always use text, maybe we should consider alternative markup.